pkg/cloudprovider/cloudprovider.go

```diff
@@ -220,8 +219,10 @@ func (c *CloudProvider) Name() string {
 	return "aws"
 }
 
-func (c *CloudProvider) GetSupportedNodeClasses() []schema.GroupVersionKind {
-	return []schema.GroupVersionKind{object.GVK(&v1beta1.EC2NodeClass{})}
+func (c *CloudProvider) GetSupportedNodeClasses() []status.Object {
+	nodeClasses := make([]status.Object, 1)
+	nodeClasses[0] = status.Object(&v1beta1.EC2NodeClass{})
+	return nodeClasses
```

Comment:
```suggestion
	return []status.Object{&v1beta1.EC2NodeClass{}}
```

You can collapse all of this